docs/source/AdministratorGuide/Resources/proxyprovider.rst

```diff
@@ -0,0 +1,85 @@
+.. _resourcesProxyProvider:
+
+==============
+ProxyProvider
+==============
+
+This resource type provide interface to obtain proxy certificate using user indentifier. The following proxy providers are represented here: ``DIRACCA``, ``PUSP``.
+
+
+DIRACCA proxy provider
+----------------------
+
+ProxyProvider implementation for the proxy generation using local CA credentials. This class is a simple CA, it's main purpose is to generate a simple proxy on the fly for DIRAC users who do not have any certificate register. To use it need to have CA certificate and key localy. Required parameters in the DIRAC configuration for its implementation::
```

Comment:
```suggestion
ProxyProvider implementation for the proxy generation using local Certification Authority (CA) credentials. This class is a simple CA, its main purpose is to generate a proxy on the fly for DIRAC users who do not have a certificate registered in DIRAC registry. To use it need to have CA certificate and key locally. Required parameters in the DIRAC configuration for its implementation::
```

But TBH I am not fully sure I understand what it does...?

Reply:
Thanks for your corrections, I'll think of a way to explain more clearly here.